pkgs/tools/graphics/vulkan-helper/default.nix

```diff
@@ -22,6 +23,16 @@ rustPlatform.buildRustPackage rec {
     addOpenGLRunpath
   ];
 
+  patches = [
+    # Fixes build on aarch64. Can be removed as soon as this PR is merged:
+    # https://github.com/imLinguin/vulkan-helper-rs/pull/3
+    (fetchpatch {
+      name = "fix-aarch64-build.patch";
+      url = "https://github.com/imLinguin/vulkan-helper-rs/commit/577400410cb7da220ab2e91c3a5ba861ccf2feca.patch";
```

Comment:
I think for an open upstream PR, we should vendor the patch (download and commit it alongside the `default.nix` file), because if at any point the PR branch is rebased, which is not uncommon, the commit will be garbage collected and `fetchpatch` will fail to build.

Reply:
I didn´t consider that, thanks!

I've force-pushed an according change: https://github.com/NixOS/nixpkgs/commit/68491006e31b1126a30bf4a61d7996fdb3db2280